biomaj/bank.py

```diff
@@ -525,10 +527,36 @@ def save_session(self):
 
         self.bank = self.banks.find_one({'name': self.name})
 
+    def format_release(self, release, reverse=False):
+        """
+        Check the release does not contains any dot '.' character that should make MongoDB complain
+
+        :param release: release number to check
+        :type release: str
+        :param reverse: Do the reverse replacement
+        :type reverse: bool
+        :return: Release string with dot '.' replaced with character set in config (release.replace)
+        :rtype: str
+        """
+        if not self.config.get('release.replacedot'):
```

Comment:
I've made a few comments on code

Le mer. 27 avr. 2016 à 15:19, Emmanuel Quevillon <notifications@github.com>
a écrit :

> In biomaj/bank.py
> <https://github.com/genouest/biomaj/pull/44#discussion_r61254896>:
>
> > @@ -525,10 +527,36 @@ def save_session(self):
> >
> >          self.bank = self.banks.find_one({'name': self.name})
> >
> > +    def format_release(self, release, reverse=False):
> > +        """
> > +        Check the release does not contains any dot '.' character that should make MongoDB complain
> > +
> > +        :param release: release number to check
> > +        :type release: str
> > +        :param reverse: Do the reverse replacement
> > +        :type reverse: bool
> > +        :return: Release string with dot '.' replaced with character set in config (release.replace)
> > +        :rtype: str
> > +        """
> > +        if not self.config.get('release.replacedot'):
>
> I've opened a new pull request
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/genouest/biomaj/pull/44/files/d235f544d677e718229ab04b92229485fa50ca28#r61254896>
>